dev/PushNotifications/PushNotificationChannel.cpp

```diff
@@ -148,6 +183,44 @@ namespace winrt::Microsoft::Windows::PushNotifications::implementation
     }
     CATCH_RETURN()
 
+    HRESULT __stdcall PushNotificationChannel::OnRawNotificationReceived(unsigned int payloadLength, _In_ byte* payload, _In_ HSTRING /*correlationVector */) noexcept
+    {
+        try
+        {
+            BOOL foregroundHandled = false;
+            THROW_IF_FAILED(InvokeAll(payloadLength, payload, &foregroundHandled));
+            THROW_HR(E_INVALIDARG);
+        }
+        catch(...)
+        {
+            //TODO: Capture the HRESULT in telemetry
+            winrt::com_array<uint8_t> payloadArray{ payload, payload + (payloadLength * sizeof(uint8_t)) };
```

Comment:
This is all shared code between the LRP and WinSDK. So it should be defined in a common header in a utility function.

Reply:
For now I have moved this code into externs.h which the files in PushNotifications/ are using. I need to see how I can make both LRP and WinSDK use the same code when daniel's changes and this PR merges into WNP. It makes my life easier then. I will take this as a bug: https://github.com/microsoft/WindowsAppSDK/issues/1481

Thanks.